lib/AppInfo/Application.php

```diff
@@ -44,6 +44,7 @@
 use OCA\Mail\Service\AvatarService;
 use OCA\Mail\Service\Group\IGroupService;
 use OCA\Mail\Service\Group\NextcloudGroupService;
+use OCA\Mail\Service\Group\ContactsGroupService;
```

Comment:
actually not used here. you may also remove the two other group service imports

Reply:
I do not yet get everything about use, but if its like imports it seems to me that since https://github.com/nextcloud/mail/commit/d18ef16578dc34ffd67e48bc5c61654e55461ce9 when BootstrapSingleton was created only App is really used? And everything else was moved to Bootstrap?

Reply:
Yes, that is absolutely correct :)

![Bildschirmfoto von 2020-01-08 10-54-28](https://user-images.githubusercontent.com/1374172/71968468-490c7580-3205-11ea-9ded-4b4dac3cf946.png)

Reply:
Ok I squashed my edits into one commit and made a second one for the cleanup of the imports in applications. Do you need it on a different pull request?